src/main/java/frc/robot/subsystems/ClimbSubsystem.java

```diff
@@ -22,237 +21,462 @@
+
+
+      m_McTwo.getPIDCtrl().setReference(McHeightTwo, CANSparkMax.ControlType.kPosition);
+        sbMcHeightTwo.setNumber(McHeightTwo);
     }
   }
 
-  public void enableClimb() {
-    if (climbEnabbled
-    /** && !m_limitSwitch.get() */
-    ) {
-
-      joystickAxis = -m_climbJoystick.getRawAxis(Constants.leftJoystickY);
-      if (joystickAxis > 0.1 || joystickAxis < -0.1) {
-        if (joystickAxis > 0) {
-          if (climbHeightOne <= Constants.climbHeightMax) {
-            climbHeightOne = climbHeightOne + (joystickAxis / 10 * 8);
+  public void highArms() {
+    if (climbEnabble) {
+      HajoystickAxis = -m_climbJoystick.getRawAxis(Constants.rightJoystickY);
+      if (HajoystickAxis > 0.1 || HajoystickAxis < -0.1) {
+        if (HajoystickAxis > 0) {
+          if (HaHeightOne + (HajoystickAxis / 10 * 8) <= Constants.HaHeightMax) {
+            HaHeightOne = HaHeightOne + (HajoystickAxis / 10 * 15);
           }
-          if (climbHeightTwo <= Constants.climbHeightMax) {
-            climbHeightTwo = climbHeightTwo + (joystickAxis / 10 * 8);
+          if (HaHeightTwo + (HajoystickAxis / 10 * 8) <= Constants.HaHeightMax) {
+            HaHeightTwo = HaHeightTwo + (HajoystickAxis / 10 * 15);
           }
         }
-        if (joystickAxis < 0) {
-          if (climbHeightOne >= 0) {
-            climbHeightOne = climbHeightOne + (joystickAxis / 10 * 6);
+        if (HajoystickAxis < 0) {
+          if (HaHeightOne + (HajoystickAxis / 10 * 6) >= 0) {
+            HaHeightOne = HaHeightOne + (HajoystickAxis / 10 * 6);
           }
-          if (climbHeightTwo >= 0) {
-            climbHeightTwo = climbHeightTwo + (joystickAxis / 10 * 6);
+          if (HaHeightTwo + (HajoystickAxis / 10 * 6) >= 0) {
+            HaHeightTwo = HaHeightTwo + (HajoystickAxis / 10 * 6);
           }
         }
       }
-      m_climbMotorControllerOne
-          .getPIDCtrl()
-          .setReference(climbHeightOne, CANSparkMax.ControlType.kPosition);
+      m_HaOne.getPIDCtrl().setReference(HaHeightOne, CANSparkMax.ControlType.kPosition);
+      sbHaHeightOne.setNumber(HaHeightOne);
 
-      m_climbMotorControllerTwo
-          .getPIDCtrl()
-          .setReference(climbHeightTwo, CANSparkMax.ControlType.kPosition);
-    } else {
-      // m_climbMotorControllerOne.getPID().setReference(0, CANSparkMax.ControlType.kVoltage);
+      m_HaTwo.getPIDCtrl().setReference(HaHeightTwo, CANSparkMax.ControlType.kPosition);
+      sbHaHeightTwo.setNumber(HaHeightTwo);
     }
   }
 
   public void periodic() {
-    if (DriverStation.isDisabled() && climbEnabbled) {
+    if (sbMcHeightOne.getDouble(0) != McHeightOne) {
+      McHeightOne = sbMcHeightOne.getDouble(0);
+    }
+
+    if (sbMcHeightTwo.getDouble(0) != McHeightTwo) {
+      McHeightTwo = sbMcHeightTwo.getDouble(0);
+    }
+
+    sbClimbEnabble.setBoolean(climbEnabble);
+  }
+
+  public void debugPeriodic() {
+    if (DriverStation.isDisabled() && climbEnabble) {
       climbEnable();
     }
```

Comment:
This needs to be in periodic as well, right?

Reply:
Yes you are correct and it is now fixed